script/Ciel/Day 14/14_01_CIEL14_1B.txt

```diff
@@ -521,82 +566,95 @@ I see...
 [sha:3333b67da81029d65792f6d02ade8823fdbc559f]{
 -- Page 77, Offset 37240.
 -- 思えば、自分を殺しかねない頭痛はあの時から始まった。
-Thinking back, that's when the headaches I couldn't kill started happening.
+Thinking back on it,
+ that's about when I started getting the headaches that felt like they could kill me.
 }
 [sha:ec0cd6621951a104ce346ac6723eaad865a9649e]{
 -- Page 77, Offset 37241.
 -- 既に俺に転移していたロアは、俺がアルクェイドに心を許す度に反発してやがったのか。
-Roa, who had already transferred inside me, must have lashed out every time I let my guard down around Arcueid.
+Roa had already transferred into me at that point,
+ so that must have been him lashing out every time I let my guard down around Arcueid.
 }
 [sha:56829aaefea281ee221943880fbe4548524ad607]{
 -- Page 78, Offset 37242.
 -- 「……そうかよ。じゃあアレか。死者狩りはもののついでで、本命は俺の監視だったのか」
-"...Is that so? Then what? You were hunting the Dead as an excuse, but you were really just monitoring me this whole time?"
+"I get it...
+ So that's what you were really up to, right?
+ Hunting the Dead was just a sideshow,
+ your real goal was to keep me under observation?"
 }
 [sha:87fb18853e06c82485ad93b62677bb35142d68f8]{
 -- Page 78, Offset 37243.
 -- 「？　……あ、そっか。そういうコトになるのね、志貴から見ると。観察って意味じゃ合ってるから、それでいっか」
-"Hm? ...Ah, right. I guess that's how you would see it. Monitoring seems to fit, so I guess we could call it that."
+// ? actually voiced
+"Hm? Ah, right...
+ I guess it makes sense you'd see it that way.
+ Observation certainly fits, so yeah, let's call it that."
 }
 [sha:cb6ab8b54d988a75f1d4fe1d9a6acebfc277f64d]{
 -- Page 79, Offset 37244.
 -- 「色々な意味で貴方は希少だったから。
-"You were a very special case in a lot of ways.
+"You see, you were a special case in a number of ways.
 }
 [sha:1f499e5f3e7a52e617178e36b76fa026039421e2]{
 -- Page 79, Offset 37245.
 -- ロアはここにきて法則を変えた。次の時代に移るのではなく、この時代に留まって“永続”しようとした。
-The rules have changed ever since Roa came here. Instead of moving to the next generation, he chose to stay in this one and become 'permanent'.
+Roa changed his rules after he came here.
+ Instead of moving on to the next generation, he chose to stay in this era and become 'permanent'.
 }
 [sha:f84133ca721ea1654d09f2e3098e400b436349d8]{
 -- Page 79, Offset 37246.
 -- 今まで何回も繰り返してきたパターンを放棄したのよ。それが罠かどうか、見極める時間もほしかった」
-He gave up on the pattern he's repeated many times until now. Although I would have liked more time to see whether it was a trap."
+He abandoned the pattern he's repeated time and time again at this point.
+ I'd hoped for more time to determine whether it was a trap or not."
```

Comment:
I agree, but I think the current one says that more clearly - assuming それが is referring to Roa sticking around, current ver makes sense to me as "I wish I had more time to establish whether this change in behaviour was a trap, but alas". Suggestion makes it sound like the time/trap were something else.

Reply:
Ah, sorry, I was unclear when I wrote this. I meant the English sounded like that, which I thought was wrong. lol

Reply:
Wait, so is this a change or not? I read それが罠かどうか as referring to him breaking the pattern

Reply:
It's a change, but I realize it doesn't come across right because of another problem


37244
The opening line needs to indicate why she monitored him, so that this line (37246) makes sense
>I did it because you were a special case in a number of way.
>...
>I also just wanted more time to...